memoized_person_or_org_data.txt

```diff
@@ -1 +1 @@
-{"Tester, Mock"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Mock", "family_name"=>"Tester", "identifiers"=>{:scheme=>"orcid", :identifier=>"1234-5678-9123-4567"}}}, "User, Contributor"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Contributor", "family_name"=>"User", :role=>"other"}}, "Laster, Firston"=>{:person_or_org=>{"type"=>"personal", "given_name"=>"Firston", "family_name"=>"Laster"}}, "Unknown"=>{:person_or_org=>{"name"=>"Unknown", "type"=>"organisational"}}, "Creator not identified."=>{:person_or_org=>{"name"=>"Creator not identified.", "type"=>"organisational"}}, "Galter Health Sciences Library"=>{:person_or_org=>{"name"=>"Galter Health Sciences Library", "type"=>"organisational"}}, "Firston Laster"=>{:person_or_org=>{"name"=>"Firston Laster", "type"=>"organisational"}}}
+{"Tester, Mock"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Mock", "family_name"=>"Tester", "identifiers"=>{:scheme=>"orcid", :identifier=>"1234-5678-9123-4567"}}}, "User, Contributor"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Contributor", "family_name"=>"User"}}, "Laster, Firston"=>{:person_or_org=>{"type"=>"personal", "given_name"=>"Firston", "family_name"=>"Laster"}}, "Firston Laster"=>{:person_or_org=>{"name"=>"Firston Laster", "type"=>"organisational"}}, "Unknown"=>{:person_or_org=>{"name"=>"Unknown", "type"=>"organisational"}}, "Creator not identified."=>{:person_or_org=>{"name"=>"Creator not identified.", "type"=>"organisational"}}, "Galter Health Sciences Library"=>{:person_or_org=>{"name"=>"Galter Health Sciences Library", "type"=>"organisational"}}}
```

Comment:
It's hard to see, but because of the change in the structure the memoization needed to change as well or specs would just keep failing.